### ISSUE TITLE
Embedded videos broken in "Animations In VCS"

### ISSUE THREAD

**Issue by jypeter** [CDAT/cdat.github.io#194]
I have opened the [Animations In VCS](https://cdat-vcs.readthedocs.io/en/latest/Jupyter/Animations.html) page in FF, Chrome and IE, but there are only error messages displayed where the videos should be

**Comment by doutriaux1** [comment]
@jypeter yes readthedoc does not regenerate the notebooks, i need to figure out how to manually update them. Thx for reporting.
